include/openenclave/internal/sgx/plugin.h

```diff
@@ -89,11 +89,57 @@ oe_result_t oe_sgx_hash_custom_claims_buffer(
  * @param[in] claims_length The length of claims.
  * @retval OE_OK on success, otherwise an appropriate error code.
```

Comment:
```suggestion
 * @retval OE_OK The operation was successful.
 * @retval other An appropriate error code.
```
Same in 107 etc.  Doxygen style with retval is that the description applies (just) to the first token after retval